core/store/src/db.rs

```diff
@@ -201,9 +203,6 @@ pub enum DBCol {
     ColStateChangesForSplitStates = 49,
 }
 
-// Do not move this line from enum DBCol
-pub const NUM_COLS: usize = 50;
```

Comment:
Seems better if this line is last? So that it's next to `ColStateChangesForSplitStates = 49,`. In the current state, note that the "This enum holds" documentation comment is misplaced.